build/MUX-Release.yml

```diff
@@ -40,6 +40,19 @@ jobs:
     publishDir : $(Build.ArtifactStagingDirectory)
 
   steps:
+
+  - task: DownloadPackage@1
+    displayName: 'Download Download Microsoft.Telemetry.Inbox.Native'
+    inputs:
+      feed: '/3415933f-ac0d-4766-8c0a-3f4c247c25f5'
+      view: 'ef61a1c1-003b-4a27-bde5-beec8301021b'
+      definition: '2fe60c09-c66f-4275-ae2d-f015c7170c72'
+      version: '10.0.18362.1-190318-1202.19h1-release.amd64fre'
+      downloadPath: '$(System.DefaultWorkingDirectory)'
+
+  - script: 'move /Y build\native\inc\MicrosoftTelemetry.h dev\telemetry\'
```

Comment:
It might be clearer to do a fully qualified path like:
move /Y $(System.DefaultWorkingDirectory)\build\native\inc\MicrosoftTelemetry.h dev\telemetry\